### ISSUE TITLE
Male emoticons being displayed as female

### ISSUE THREAD

**Issue by GSPP** [signalapp/Signal-Android#9883]
When using certain male emoticons on Signal Android these get displayed as female. Signal Desktop correctly displays them as male.

Android:

![image](https://user-images.githubusercontent.com/12032350/88795462-59377280-d1a0-11ea-8091-87379a3fa67b.png)

Desktop:

![image](https://user-images.githubusercontent.com/12032350/88795341-29886a80-d1a0-11ea-989b-61451ea23652.png)

**Comment by greyson-signal** [comment]
Looks like what you're seeing on desktop isn't the [male version](https://emojipedia.org/man-getting-haircut/), it's the [gender-neutral version](https://emojipedia.org/person-getting-haircut/). Seems to be that desktop has more of the gender-neutral emojis, but android is either missing them or has the female glyph mapped to the same codepoint. The gender-neutral glyph is newer I think, as the [emojipedia article](https://emojipedia.org/woman-getting-haircut/) seems to imply that the female and gender-neutral varieties often share the same glyph.

I'll have to see if there's any updates in our emoji library that can address this.

**Comment by GSPP** [comment]
I held down the haircut icon in the drawer. This opened a menu for man/woman/person. If I select the man there I get the woman icon immediately in the message text box. The emoticon drawer displays the icon as a man, though.

**Comment by greyson-signal** [comment]
If you 'held down the haircut icon', I'm guessing you did this on mobile, yes? And because holding down the icon showed multiple gender options, I'm assuming that you're using your system keyboard emoji selector, since our in-app emoji selector does not do that.

This is Gboard. From top to bottom, it's 'person', 'man', 'woman'. I typed them in that order. I see the 'person' glyph is mapped to the 'woman' glyph in Signal, and that's the problem we discussed. Are you seeing different behavior?

<img src="https://user-images.githubusercontent.com/37311915/89046375-dedf2d80-d31a-11ea-8a80-7ea70cb775c2.jpeg" width="500" />
